configuration/openmrs/i18n/clinical/locale_en.json

```diff
@@ -87,6 +87,7 @@
 
   "CLINICAL_ORDER_RADIOLOGY_NEED_PRINT": "Need Print for this order.",
   "CLINICAL_ORDER_RADIOLOGY_NEED_PRINT_BUTTON": "Needs Print",
+  "CLINICAL_ACCEPT": "Validate",
```

Comment:
Could you change this key to [this](https://github.com/mekomsolutions/bahmni-config-haiti/search?q=CLINICAL_ACCEPT)

Reply:
Done